scripts/postgres/OpenRegistry.sql

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
@guacamole do we want to keep this? since we're already using `migrate` to handle all of this, maybe we can remove it?

Reply:
let's keep it since this is our working schema